recipe/meta.yaml

```diff
@@ -11,12 +11,12 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 0
   noarch: python
   script: python setup.py install --prefix ${PREFIX}
```

Comment:
this has to change. 
`script` section is probably not required.

look at #9 I issued to get inspiration.

Reply:
I made all of these changes!